pkgs/development/python-modules/grpcio-tools/default.nix

```diff
@@ -2,15 +2,20 @@
 
 buildPythonPackage rec {
   pname = "grpcio-tools";
-  version = "1.48.1";
+  version = "1.49.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1178f2ea531f80cc2027ec64728df6ffc8e98cf1df61652a496eafd612127183";
+    sha256 = "sha256-hMxk5bRrrUPV170v13K2VuugNmlhGHqEfpCOLLc125E=";
   };
 
   outputs = [ "out" "dev" ];
 
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace 'protobuf>=4.21.3,<5.0dev' 'protobuf'
+  '';
+
```

Comment:
needs a rebase, looks like it's conflicting with ccce3fcb58e000354835dda97fb0183585e2a213

Reply:
This bump has no urgency, I'm going to queue it for the next staging cycle and rebase after staging-next is in. I don't want to delay staging-next any further.

Reply:
Rebased.